Podcast/CurrentUser.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  CurrentUser.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 3/1/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class CurrentUser: User {
```

Comment:
we don't need a current user class, we should have a User and Session class, where some global class such as System will hold a reference to the current user's User and Session object